news/2024/2024-09-02-the-followpoint-dreamxiety-the-picasso-of-osu!.md

```diff
@@ -0,0 +1,257 @@
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/13103233">Dreamxiety</a></p>
+
+Maybe tmi but completely honest answer; my crush on 9th grade played it so I downloaded it too so I could play with him. Back then I knew nothing about anime or things like that, so it was kinda weird seeing most songs in the game be Japanese songs.
+
+I also thought it was really weird that even English songs have completely unrelated anime girl background. Obviously I'm now used to it and osu! was actually the main thing that exposed me to more anime and overall Japanese related things, although I'm still not an anime watcher type of person to this day.
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/11354436" style="background-image: url('/wiki/shared/news/2024-09-02-the-followpoint-dreamxiety-the-picasso-of-osu!/avatar-Utiba.jpeg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/11354436">Utiba</a></p>
+
+What made you want to do art in the first place?
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/13103233" style="background-image: url('/wiki/shared/news/2024-09-02-the-followpoint-dreamxiety-the-picasso-of-osu!/avatar-Dreamxiety.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/13103233">Dreamxiety</a></p>
+
+I don't know, it's just something I did when I was a child and I never stopped since then. Many children like drawing but drop it as they grow up, but I didn't. I was maybe like 3 or 4 when I started and I just kept going.
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/11354436" style="background-image: url('/wiki/shared/news/2024-09-02-the-followpoint-dreamxiety-the-picasso-of-osu!/avatar-Utiba.jpeg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/11354436">Utiba</a></p>
+
+What is your drawing process like? Is there a certain workflow you follow when working on each project, or does each project have a different process?
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/13103233" style="background-image: url('/wiki/shared/news/2024-09-02-the-followpoint-dreamxiety-the-picasso-of-osu!/avatar-Dreamxiety.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/13103233">Dreamxiety</a></p>
+
+If I'm looking for ideas, I'll browse through the internet to find inspirations. For osu contests specifically, I often just search for art that has the theme. It's a pretty good method since what comes at the top of the search are the things people generally find appealing, and what people have in mind when the name of the theme comes up.
```

Comment:
```suggestion
If I'm looking for ideas, I'll browse through the internet to find inspirations. For osu! contests specifically, I often just search for art that has the theme. It's a pretty good method since what comes at the top of the search are the things people generally find appealing, and what people have in mind when the name of the theme comes up.
```